all-products.html

```diff
@@ -11,59 +11,99 @@
 		</header>
 		
 		<div class="wrapper">
+			
 			<!-- This div is for the products on the page-->
 			<div class="products">
                 <h1>All Products</h1>
 
-                <!-- Flexbox to divide each row easier-->
-                <div class="flex-container">
-                    <div class="flex-child"><p>Image will go here</p></div>
-                    <div class="flex-child"><p>Image will go here</p></div>
-                    <div class="flex-child"><p>Image will go here</p></div>
-                    <div class="flex-child"><p>Image will go here</p></div>
+                <div class="container">

```

Comment:
In this version I changed it from a flexbox to floats and specific values for box sizes